### DISCUSSION TITLE
Will you consider adding a file centric section to which key?

### DISCUSSION THREAD

**Discussion by danielo515** [LunarVim/LunarVim#2841] [General]
### Feature Description

I think it will be useful to have a dedicated file section in the default wich-key.
Space-f is kind of convenient, but it space-f-f can be almost as convenient, and having an entire section around file stuff will be useful and easier for newcomers.
Some of the actions that can be under such menu (I will not list mappings, just actions) could be:

- Yank file full path
- Yank file relative path
- Browse sibling files
- Browse parent folder
- Delete current file and kill buffer
- Yank file contents
- Execute current file
- Reload file

And I'm pretty sure the list can continue.
What do you think?

### Describe the alternatives you have considered

I have this setup in my own config, and people may do the same.

**Comment by abzcoding** [comment]
sounds like a plan, a PR would be appreciated
